worldedit-core/src/main/java/com/sk89q/worldedit/util/PropertiesConfiguration.java

```diff
@@ -57,12 +58,23 @@
      *
      * @param path the path tot he configuration
```

Comment:
Fix typo while you're here? :P
```suggestion
     * @param path the path to the configuration
```